index.html

```diff
@@ -187,5 +187,21 @@ <h3 class="conn">LET’S CONNECT</h3>
           </div>
 
     </section>
+ 
+    <section class="contact_section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-content">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/xeqwngzr" method="POST">
+            <input type="text" name="name" placeholder="Lorem Ipsum" placeholder="Name" maxlength="30" required>
+            <input type="email" name="email" placeholder="loremipsum@gmail.com" placeholder="Email" required>
+            <textarea  name="message" maxlength="500" required>
+                Write your message here...
+            </textarea>
```

Comment:
- [ ] [REQUIRED] This text on line 200 should be a placeholder and not a value of the `textarea` element. Please use the `placeholder` attribute to set the text.